public/components/query_compare/search_result/result_components/result_grid.tsx

```diff
@@ -44,17 +44,19 @@ export const ResultGridComponent = ({
     );
   };
 
-  const getDlTmpl = (doc: IDocType) => {
+  const getDlTmpl = (doc: Document) => {
+    const sourceFields = Object.assign(doc._source, doc.fields);
```

Comment:
In the event that we have the same field in the `source` and the `fields`, which one should take precedence? Or should we find a way to display both and indicate which fields are from mappings/fields and which fields are from the source?

Reply:
Based on the description in #201, I implemented this so the field from the `fields` would take precedence. I can change this so that both are displayed though.